packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useCallback, useEffect } from 'react';
 import { useActiveWalletAccount, useCreateNewRealAccount, useSettings } from '@deriv/api';
 import { toMoment } from '../../../../shared/src/utils/date';
-import { Success } from '../../features/cfd/screens/Success';
+import { CFDSuccess } from '../../features/cfd/screens/CFDSuccess';
```

Comment:
nit, we should import it from the barrel file:
```
import { CFDSuccess } from '../../features/cfd/screens
```

Reply:
Then what the purpose of exporting it from screens folder?